autogluon/training/docker/0.6/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,61 @@
+ARG PYTHON_VERSION=3.8.10
+
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/mxnet-training:1.9.0-cpu-py38-ubuntu20.04-sagemaker
+
+# PYTORCH_SKIP_CUDNN_COMPATIBILITY_CHECK - see https://github.com/autogluon/autogluon/issues/2534
```

Comment:
Thanks for adding this explanatory comment! This is very helpful to have in the dockerfile!